metricbeat/module/kubernetes/_meta/fields.yml

```diff
@@ -11,3 +11,8 @@
       description: >
         Information and statistics of pods managed by kubernetes.
       fields:
+        - name: labels.*
```

Comment:
This is already defined here: https://github.com/elastic/beats/blob/master/libbeat/processors/add_kubernetes_metadata/_meta/fields.yml#L31

Reply:
to clarify, that fields.yml is included in any beat using the processor, including Metricbeat

Reply:
oh, for some reason beyond my comprehension I tested without adding that field and it appeared as non mapped at kibana.

I'll re-check and remove if the field is already there.

thanks!

Reply:
done @exekias 
please, re-review if you will